experiments/AtmosLES/dycoms.jl

```diff
@@ -376,9 +376,9 @@ function main()
     timeend = FT(100)
```

Comment:
@simonbyrne @smarras79 What is the longest simulation `timeend` we can afford to link against the  current `CI` pipeline ?

Reply:
It's completely parallel so there's no time-to-completion cost. The current budget per run is 60 minutes.